core/clingutils/src/TClingUtils.cxx

```diff
@@ -4825,7 +4832,7 @@ ROOT::ESTLType ROOT::TMetaUtils::STLKind(const llvm::StringRef type)
 {
    static const char *stls[] =                  //container names
       {"any","vector","list", "deque","map","multimap","set","multiset","bitset",
-         "forward_list","unordered_set","unordered_multiset","unordered_map","unordered_multimap",0};
+         "forward_list","unordered_set","unordered_multiset","unordered_map","unordered_multimap", "RVec", 0};
```

Comment:
```suggestion
         "forward_list","unordered_set","unordered_multiset","unordered_map",
         "unordered_multimap", "RVec", 0};
```

Reply:
I don't understand the suggestion: the line is not beyond ROOT's 120 columns, while clang-format would actually like for the list to be formatted this way:

```cpp
   static const char *stls[] = // container names
      {"any",           "vector",
       "list",          "deque",
       "map",           "multimap",
       "set",           "multiset",
       "bitset",        "forward_list",
       "unordered_set", "unordered_multiset",
       "unordered_map", "unordered_multimap",
       "RVec",          0};
```